src/main/pages/storylineoverview_serializer.py

```diff
@@ -26,18 +26,27 @@ def get_all_storylines(self, page):
                 it_dict = {"name": it.name, "icon": it.icon}
                 it_array.append(it_dict)
 
+            thumbnail = {
+                "url": "",
+                "width": 1,
+                "height": 1,
+            }
+
+            if sl.thumbnail_rendition_url is not None:
+                thumbnail = {
+                    "url": sl.thumbnail_rendition_url.url,
+                    "width": sl.thumbnail_rendition_url.width,
+                    "height": sl.thumbnail_rendition_url.height,
+                }
```

Comment:
In hoeverre is dit nodig? Ik zie namelijk dat de frontend al checkt of er een thumbnail is en verder niets doet met de width en height properties.